docs/vNext/Fabric_Configuration_New.md

```diff
@@ -0,0 +1,1115 @@
+>> * Cannot specify to use an admin identity in a workload request using the `#ORG2` type notation
+
+How you select which implementation you use is via the provided network configuration file and specifically the version you declare in that file
+
+> * a version of 1.0 selects the legacy connector
+> * a version of 2.0.0 selects the new connector
+
+For example
+
+```yaml
+name: Fabric
+version: "1.0"
+```
+would select the legacy connector and the file must conform to the old deprecated network configuration file format
+
+```yaml
+name: Fabric
+version: "2.0.0"
+```
+
+would select the new connector and the file must conform to the new network configuration file format
+
+
+## Overview
+
+This page introduces the Fabric adapter that utilizes the Common Connection Profile (CCP) feature of the Fabric SDK to provide compatibility and a unified programming model across different Fabric versions.
+
+> The latest supported version of Hyperledger Fabric is v2.x
+
+The adapter exposes many SDK features directly to the user callback modules, making it possible to implement complex scenarios.
+
+> Some highlights of the provided features:
+> * supporting multiple orderers
+> * supporting multiple channels and contracts
+> * metadata and private collection support for contracts
+> * support for TLS and mutual TLS communication
+> * option to select the identity for submitting a TX/query
+> * detailed execution data for every TX
+
+## Installing dependencies
+
+You must bind Caliper to a specific Fabric SDK to target the corresponding (or compatible) SUT version. Refer to the [binding documentation](./Installing_Caliper.md#the-bind-command) for details. It is confirmed that a 1.4 Fabric SDK is compatible with a Fabric 2.1 and 2.2 SUT however any installation or instantiation of contracts (chaincode) will only work if you are still using the old lifecycle implementation.
+
+### Binding with Fabric SDK 2.x
+> Note that when using the binding target for the Fabric SDK 2.x there are capability restrictions:
+> * The 2.x SDK does not facilitate administration actions. It it not possible to create/join channels, nor install/instantiate contract. Consequently the 2.x binding only facilitates operation with a `--caliper-flow-only-test` flag
+> * The 2.x SDK only supports operation using a `gateway`. Consequently the 2.x binding requires a `--caliper-fabric-gateway-enabled` flag
+
+### Using the Operational capabilities of the Adapters
+The ability to create channels and perform install/instantiate of contracts (chaincode) remains in the new 1.x sdk based adapters, and nothing equivalent exists for the 2.x sdk based adapters. This capability remains mainly to support the caliper integration tests and the use of these mechanisms to setup your channels and contracts is highly discouraged. This capability is considered legacy and will not be maintained. You should look into alternative methods for setting up your network prior to running caliper to benchmark it.
```

Comment:
Upper case for Caliper, probably worth a global search/replace (won't mention other occurrences in further review comments)

Reply:
Only found a couple to be done which were in this section.